executor/tiflash_test.go

```diff
@@ -622,20 +626,22 @@ func (s *tiflashTestSuite) TestDispatchTaskRetry(c *C) {
 	tk.MustExec("insert into t values(2,0)")
 	tk.MustExec("insert into t values(3,0)")
 	tk.MustExec("insert into t values(4,0)")
-	tb := testGetTableByName(c, tk.Se, "test", "t")
-	err := domain.GetDomain(tk.Se).DDL().UpdateTableReplicaInfo(tk.Se, tb.Meta().ID, true)
-	c.Assert(err, IsNil)
+	tb := tk.GetTableByName("test", "t")
+	err := domain.GetDomain(tk.Session()).DDL().UpdateTableReplicaInfo(tk.Session(), tb.Meta().ID, true)
+	require.NoError(t, err)
 	tk.MustExec("set @@session.tidb_enforce_mpp=ON")
-	c.Assert(failpoint.Enable("github.com/pingcap/tidb/store/mockstore/unistore/mppDispatchTimeout", "3*return(true)"), IsNil)
+	require.Nil(t, failpoint.Enable("github.com/pingcap/tidb/store/mockstore/unistore/mppDispatchTimeout", "3*return(true)"))
 	tk.MustQuery("select count(*) from t").Check(testkit.Rows("4"))
-	c.Assert(failpoint.Disable("github.com/pingcap/tidb/store/mockstore/unistore/mppDispatchTimeout"), IsNil)
+	require.Nil(t, failpoint.Disable("github.com/pingcap/tidb/store/mockstore/unistore/mppDispatchTimeout"))
 }
 
-func (s *tiflashTestSuite) TestCancelMppTasks(c *C) {
+func TestCancelMppTasks(t *testing.T) {
 	testleak.BeforeTest()
-	defer testleak.AfterTest(c)()
+	defer testleak.AfterTestT(t)()
```

Comment:
goroutine leaks checking already covered by `goleak` in `main_test.go`.

Reply:
ok, get it. will fix soon